### ISSUE TITLE
Setup failed for porthutv: integration not found

### ISSUE THREAD

**Issue by KixMan28** [vorostamas/portHuTV-homeassistant#16]
## Version of the custom_component
 v1.0.5

## Configuration

```yaml

## System Health

version | core-2021.6.2
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
virtualenv | false
python_version | 3.8.9
os_name | Linux
os_version | 5.4.83-v7
arch | armv7l
timezone | Europe/Budapest

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
Github API Calls Remaining | 4873
Installed Version | 1.12.4
Stage | running
Available Repositories | 823
Installed Repositories | 18

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | June 17, 2021, 2:00 AM
relayer_connected | true
remote_enabled | true
remote_connected | true
alexa_enabled | true
google_enabled | false
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | failed to load: timeout

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 5.13
-- | --
update_channel | stable
supervisor_version | supervisor-2021.05.4
docker_version | 19.03.15
disk_total | 13.9 GB
disk_used | 8.7 GB
healthy | true
supported | true
board | rpi3
supervisor_api | ok
version_api | ok
installed_addons | Samba share (9.5.0), Duck DNS (1.12.5), Mosquitto broker (6.0.1), File editor (5.3.1), SSH & Web Terminal (8.2.3), MariaDB (2.3.0), WireGuard (0.5.1), Zigbee2mqtt Edge (edge), ESPHome (1.18.0), Simple Scheduler (0.22)

</details>

<details><summary>Lovelace</summary>

dashboards | 4
-- | --
resources | 8
views | 18
mode | storage

</details>

```

## Describe the bug

The integration fails to start, the entities are unavailable.

## Debug log

```text

Logger: homeassistant.setup
Source: setup.py:172
First occurred: 7:20:57 AM (1 occurrences)
Last logged: 7:20:57 AM

Setup failed for porthutv: Integration not found.

```

**Comment by vorostamas** [comment]
Thanks for reporting!
This issue was fixed in #15, please update to v1.0.6

**Comment by KixMan28** [comment]
Thank you.
